lib/transbank/sdk/onepay/models/shopping_cart.rb

```diff
@@ -24,6 +24,10 @@ def initialize(items = [])
       # @param item [Item] an instance of [Item]
       # @return [boolean] return true if item is successfully added
       def add(item)
+        new_total = total + item.amount * item.quantity
```

Comment:
You can use item.total
https://github.com/TransbankDevelopers/transbank-sdk-ruby/blob/master/lib/transbank/sdk/onepay/models/item.rb#L83